keyboards/40percentclub/polyandry/keymaps/ryjelsum/keymap.c

```diff
@@ -0,0 +1,27 @@
+/* Copyright 2021
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	LAYOUT(
+		KC_1, KC_2, KC_3, 
+		KC_4, KC_5, KC_6, 
+		KC_7, KC_8, KC_9, 
+		KC_0, KC_A, KC_B
+	),
```

Comment:
This `ryjelsum` keymap seems to be exactly the same as the `default` keymap — it's not useful in this state and should just be removed from the PR.

Reply:
Totally correct that it's the same, shouldn't have been in the PR in the first place actually.